Detectors/EMCAL/base/include/EMCALBase/Geometry.h

```diff
@@ -9,8 +9,8 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
-#ifndef ALICEO2_EMCAL_GEOMETRY_H_
-#define ALICEO2_EMCAL_GEOMETRY_H_
+#ifndef DETECTORS_EMCAL_BASE_INCLUDE_EMCALBASE_GEOMETRY_H_
+#define DETECTORS_EMCAL_BASE_INCLUDE_EMCALBASE_GEOMETRY_H_
```

Comment:
why this change? (It is generally recommended to *not change* parts of the code *not* related to a new feature ... Amongst others, this often causes conflicts when trying to cherry-pick commits simply because code changes are less local and touch more lines.)

Reply:
I made the changes recommended by the cpplint and O2linter. I thought while I am already adding something I could fix th warnings.
If you think it's a problem I can change it.